frontend/src/index.html

```diff
@@ -155,7 +161,9 @@
                   <!-- First line -->
                   <div class="flex items-start justify-between space-x-1">
                     <!-- Title -->
-                    <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.highlight}} { "attribute": "title" } {{/helpers.highlight}}</h3>
+                    {{#_highlightResult.title}}
+                    <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{{value}}}</h3>
+                    {{/_highlightResult.title}}
```

Comment:
Using the `helpers.highlight` function instead of `#_highlightResult` has some advantage in terms of formatting the HTML by prettier and bundling by Parcel. It's a long story.

Reply:
i am using custom appInfiniteHits to render hits. if you check that widget i am using hogan.js to compile this item template. and while compiling existing code is not compatible with that, that's why i had to change to current code.

Reply:
This is the point: you were not supposed to introduce a custom `appInfiniteHits` in the first place :)